### PR TITLE
Refactor pacman eat dotand move

### PR DESCRIPTION
**Pull request by MirkoFer-622**
Refactoring of code to change origin of functions since DrawComponents to other classes that don't need external Libraries.

### CODE REVIEW

File: src/main/java/pacman/Creator.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import java.util.ArrayList;
+public class Creator {
+
+    private static final int CANT_GHOST = 4;
+    public static final int POS_GHOST_1_X = 30;
+    public static final int POS_GHOST_2_Y = 180;
+    public static final int POS_GHOST_3_Y = 60;
+    public static final int POS_GHOST_4_X = 210;
+    private ArrayList<Ghost> ghosts = new ArrayList<Ghost>();
+    public ArrayList<Ghost> createGhots() {
```
Comment (Laura4lilavati) [inline src/main/java/pacman/Creator.java]: Please change the name of the method to CreateGhost()

File: src/main/java/pacman/DrawComponents.java [hunk 2]
```diff
@@ -40,10 +32,10 @@
     private int x = pacman.getX();
     private int y = pacman.getY();
     private String direction;
-    private static final int CANT_GHOST = 4;
     private static final int TIME_GHOST_IS_EATABLE = 3;
     private ArrayList<Position> walls;
     private ArrayList<Dot> dots;
+    private Creator Creator;
```
Comment (Laura4lilavati) [inline src/main/java/pacman/DrawComponents.java]: Please change the name of the object to creator.

**Comment by J0c4** [comment]
Hi @MirkoFer-622 , thanks for pushing this PR, it looks like it has some checkStyle issues, please check and push another commit to fix them, thanks!

File: src/main/java/pacman/Creator.java (hunk 1 above)
Comment (J0c4) [reply src/main/java/pacman/Creator.java]: agree, but we should keep camel case though: createGhost()

File: src/main/java/pacman/ActionKeyPacman.java [hunk 3]
```diff
@@ -20,17 +24,17 @@ public ActionKeyPacman(final DrawComponents dComponents) {
      */
     @Override
         public void actionPerformed(final ActionEvent event) {
-            if (keyRight) {
-                drawComponents.moveRight();
-            }
             if (keyLeft) {
-                drawComponents.moveLeft();
+                drawComponents.move(NUMBER_ZERO);
```
Comment (samuelloza) [inline src/main/java/pacman/ActionKeyPacman.java]: mmm I think can be more descriptive,  for example MOVE_LEFT

File: src/main/java/pacman/ActionKeyPacman.java [hunk 4]
```diff
@@ -20,17 +24,17 @@ public ActionKeyPacman(final DrawComponents dComponents) {
...
+            }
+            if (keyRight) {
+                drawComponents.move(NUMBER_ONE);
             }
             if (keyUp) {
-                drawComponents.moveUp();
+                drawComponents.move(NUMBER_TWO);
```
Comment (samuelloza) [inline src/main/java/pacman/ActionKeyPacman.java]: Same here

File: src/main/java/pacman/ActionKeyPacman.java [hunk 5]
```diff
@@ -20,17 +24,17 @@ public ActionKeyPacman(final DrawComponents dComponents) {
...
             }
             if (keyDown) {
-                drawComponents.moveDown();
+                drawComponents.move(NUMBER_THREE);
```
Comment (samuelloza) [inline src/main/java/pacman/ActionKeyPacman.java]: Same here

**Comment by julia-jpg** [comment]
everything ok

**Comment by MirkoFer-622** [comment]
I think, everithing OK!

**Pull request merged**